pydy/codegen/ode_function_generators.py

```diff
@@ -14,6 +16,13 @@
     from sympy.printing.theanocode import theano_function
 
 from .cython_code import CythonMatrixGenerator
+import pydy
+
+
+class PyDyDepWarning(DeprecationWarning):
```

Comment:
Maybe we should make this `PyDyDeprecationWarning`, sorry to imply the other earlier.